src/main/bindings/c/bindings.h

```diff
@@ -201,6 +226,8 @@ void config_iterHosts(const struct ConfigOptions *config,
                       void (*f)(const char*, const struct ConfigOptions*, const struct HostOptions*, void*),
                       void *data);
 
+uint32_t config_getNHosts(const struct ConfigOptions *config);
+
```

Comment:
This doesn't seem to be used from C anywhere. Do you plan to use this later, or can we avoid exporting it?

Reply:
No specific plans, but it doesn't hurt; IMO it makes sense to just be consistent with the other definitions here.